src/isomorphic/classic/element/ReactElementValidator.js

```diff
@@ -184,7 +184,8 @@ function validatePropTypes(element) {
 var ReactElementValidator = {
 
   createElement: function(type, props, children) {
-    var validType = typeof type === 'string' || typeof type === 'function';
+    var validType = typeof type === 'string' || typeof type === 'function' ||
+                    (type !== null && typeof type === 'object');
```

Comment:
Actually I guess this is a minor-version change as it allows objects as element types. Should we maybe do a backport of this and exclude this change? We might also be able to get away with not cherry-picking this at all…